charts/seed-monitoring/charts/core/charts/prometheus/templates/config.yaml

```diff
@@ -627,6 +627,28 @@ data:
         replacement: 'shoot_node_logging_incoming_$1'
 {{- end }}
 
+{{- if .Values.reversedVPN.enabled }}
+    - job_name: reversed-vpn-envoy-side-car
+      honor_labels: false
+      kubernetes_sd_configs:
+      - role: service
+        namespaces:
+          names: [{{ .Release.Namespace }}]
+      relabel_configs:
+      - source_labels:
+        - __meta_kubernetes_service_name
+        - __meta_kubernetes_service_port_name
+        action: keep
+        regex: vpn-seed-server;metrics
+      metric_relabel_configs:
+      - source_labels: [__name__]
+        target_label: __name__
+        regex: '(.*)'
+        action: replace
+        replacement: 'reversed_vpn_$1'
```

Comment:
```suggestion
```
In prometheus it is idiomatic to keep the original metric name and just use labels for further differentiation. You could use the default `job` label to filter for the envoy metrics of your own deployment.

Reply:
Adapted accordingly with Sebastian's latest commit.